demo/sections/components/Combobox.vue

```diff
@@ -106,11 +106,9 @@
 </script>
 
 <style>
-  .combobox__demo {
-    @apply
-    h-80
-    flex
-    flex-col
-    gap-4
-  }
+.combobox__demo { @apply
+  flex
+  flex-col
+  gap-4
+}
```

Comment:
I removed `h-80` from here. 

| **Before** | **After** |
| ---- | ---- |
| ![image](https://github.com/PrefectHQ/prefect-design/assets/22418768/a9a29178-3561-456f-8025-ece09b6d4da8) | ![image](https://github.com/PrefectHQ/prefect-design/assets/22418768/a9ebfefd-4f91-43c4-9813-d48adb5bca43) |

Why? 
I didn't even notice there were more demos further down that i had to scroll to.. also more consistent w/ p-select demo page.